test.sh

```diff
@@ -19,6 +19,28 @@ for dir in ${VERSIONS}; do
     VERSION=$dir test/run
   fi
 
+  if [ -n "${TEST_CONU_MODE}" ]; then
+    if [[ -x test/run-conu ]]; then
+      docker run \
+        --net=host \
+        -e VERSION=$dir \
+        -e IMAGE_NAME \
+        --rm -v /dev:/dev:ro \
+        -v /var/lib/docker:/var/lib/docker:ro \
+        --security-opt label=disable \
+        --cap-add SYS_ADMIN \
+        -ti \
+        -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
Hmpfs, this means we give _a lot_ of trust into that container.

Reply:
what's the difference in running the tests directly on host?

Reply:
I hope we didn't run privileged container, nor UID=0 container in testsuite so far in our suite -- except for build -- but I don't know, I thought `docker build` isn't mounting insecure things into the container.  That said, I thought it is banned policy to give a random container write access to a docker socket.  But I'm probably too old school;  /me just dropped the UID from docker group, and I'll run the suite in VM.